index.html

```diff
@@ -229,38 +229,71 @@ <h3>
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
               </p>
-
-              <p class="draft-status"><b>Draft state:</b>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
+              </p>
+              <p class="input-documents"><b>Cryptosuites:</b>
+                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature</a>,
+                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">EdDSA</a>,
+                <a href="https://w3c-ccg.github.io/di-ecdsa-secpr1-2019/">NIST ECDSA</a>,
+                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Koblitz ECDSA</a>
+              </p>
               <p class="milestone"><b>Expected completion:</b> <span class=todo>WG-START+24 months</span></p>
             </dd>
           </dl>
 
         </section>
 
+        <section id="normative">
+          <h3>
+            CCG and IETF Specifications
+          </h3>
+          <p>
+Depending on progress in the
+<a href="https://www.w3.org/community/credentials/">W3C Credentials Community Group</a>
+and the <a href="https://www.ietf.org/">IETF</a>, the Working Group may also
+produce W3C Recommendations for the following documents:
+          </p>
+
+          <table>
+            <tr>
+              <th>Specification</th>
+              <th>Description</th>
+            </tr>
+            <tr>
+              <td>
+<a href="https://or13.github.io/lds-pgp2021/">PGP&nbsp;Cryptosuite</a>
+              </td>
+              <td>
+A cryptographic digital signature suite that utilizes Pretty Good Privacy [<a
+href="https://datatracker.ietf.org/doc/html/rfc4880">RFC4880</a>].
+              </td>
+            </tr>
+            <tr>
+              <td>
+<a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+&nbsp;Cryptosuite</a>
```

Comment:
I think for BBS+, we are not potentially producing W3C recommendation, but using it as an input document once it is published in IETF, so suggest to move it to a section above with a caveat `(once it is publised in IETF)`

Reply:
I would suggesting moving this comment to an issue, and accepting this PR as is.

Reply:
I think there is confusion over what an "input document" is. An W3C input document is never an IETF RFC. An input document is some incubated specification that is intended to become a W3C Recommendation. In that sense, our input document for the BBS+ Cryptosuite is expected to be `https://w3c-ccg.github.io/ldp-bbs2020/`, since we have no other input document to point to at present.

Reply:
than VC-JWT (already a W3C recommendation) cannot be an input document. How is VC-JWT and JWS (IETF RFC) should be referred to if not as an "input document" in a W3C sense.

Reply:
> than VC-JWT (already a W3C recommendation) cannot be an input document. How is VC-JWT and JWS (IETF RFC) should be referred to if not as an "input document" in a W3C sense?

Yes, this is true. VC-JWT is a bit of an odd duck in that we defined it in the core spec in the VC 1.0 WG, and we should probably make it a stand-alone REC in the VC 2.0 WG. Ultimately, it probably doesn't matter -- it's a detail that most will fail to grasp, and the VCWG will do whatever we have consensus to do, which could be one of: 1) leave it in the VCDM spec, 2) break it out into it's own standalone spec that can evolve at its own pace, 3) something else that gets consensus in the WG. 

I think the reason we're listing it specifically is that we're trying to call out that people really want to work on and refine the VC-JWT encoding. By default, anything in the VC 1.0 spec is up for discussion/debate/refinement in the VC 2.0 work... so, we could not mention VC-JWT at all and still be fine. However, people wanted to very specifically call it out as something that they were expecting to work on.

Reply:
I may be missing the point on what needs to be referenced here but the underlying cryptographic algorithm definition for BBS in the form of an IETF draft (currently incubated at DIF) is available [here](https://github.com/decentralized-identity/bbs-signature), the intent is to propose this as a work item to the CFRG. However I do agree with manu that, there still needs to be another document that defines the proof suite in the context of a framework like linked data integrity also e.g (https://w3c-ccg.github.io/ldp-bbs2020/)

Reply:
Apologies a more accurate link to the actual hosted spec is https://identity.foundation/bbs-signature/draft-bbs-signatures.html

Reply:
> I may be missing the point on what needs to be referenced here

The document that needs to be referenced is the document that will be the basis of the W3C VCWG specification. This document: https://identity.foundation/bbs-signature/draft-bbs-signatures.html will be headed to IETF, thus is definitely not the document we should reference. Instead, it would be good for https://w3c-ccg.github.io/ldp-bbs2020/ to reference https://identity.foundation/bbs-signature/draft-bbs-signatures.html.